test/cmdlineTests/storage_layout_value_types_packed/output.json

```diff
@@ -0,0 +1,4 @@
+{"contracts":{"fileA":{"A":{"storageLayout":{"storage":[{"ast_id":"2","contract":"A","label":"x","offset":"0","slot":"0","type":"t_uint64"},{"ast_id":"4","contract":"A","label":"y","offset":"8","slot":"0","type":"t_uint128"},{"ast_id":"6","contract":"A","label":"z","offset":"0","slot":"1","type":"t_uint128"},{"ast_id":"8","contract":"A","label":"addr","offset":"0","slot":"2","type":"t_address"},{"ast_id":"12","contract":"A","label":"array","offset":"0","slot":"3","type":"t_array(t_uint256)2_storage"}],"types":{"t_address":{"encoding":"inplace","label":"address","numberOfBytes":"20","numberOfSlots":"1"},"t_array(t_uint256)2_storage":{"base":"t_uint256","encoding":"inplace","label":"uint256[2]","numberOfBytes":"32","numberOfSlots":"2"},"t_uint128":{"encoding":"inplace","label":"uint128","numberOfBytes":"16","numberOfSlots":"1"},"t_uint256":{"encoding":"inplace","label":"uint256","numberOfBytes":"32","numberOfSlots":"1"},"t_uint64":{"encoding":"inplace","label":"uint64","numberOfBytes":"8","numberOfSlots":"1"}}}}}},"errors":[{"component":"general","formattedMessage":"fileA:1:1: Warning: Source file does not specify required compiler version!
+contract A { uint64 x; uint128 y; uint128 z; address addr; uint[2] array; }
+^-------------------------------------------------------------------------^
+","message":"Source file does not specify required compiler version!","severity":"warning","sourceLocation":{"end":75,"file":"fileA","start":0},"type":"Warning"}],"sources":{"fileA":{"id":0}}}
```

Comment:
@chriseth I don't remember precisely, but is this message `Source file does not specify required compiler version!...` suppose to be removed automatically?

Reply:
yes, it should be removed.

Reply:
Ah, no, it's not, only the recommended compiler version is removed.